functions/gravityForms/getGfFieldValidationSchema.js

```diff
@@ -1,41 +1,40 @@
 import getGfFieldId from '@/functions/gravityForms/getGfFieldId'
-import * as Yup from 'yup'
-
-// TODO Refactor this file for scaling purposes.
+import StringSchemaFactory from '@/functions/gravityForms/yupSchema/StringSchemaFactory'
 
 /**
- * Map props to validation schemas.
+ * Match field type with Yup schema object.
+ *
+ * Match GravityForm field to Yup API property.
  *
- * @param {Object} fieldData GravityForm field props.
+ * @author WebDevStudios
+ * @see https://github.com/jquense/yup#api
+ * @param {Object}   fieldData GravityForm field props.
+ * @returns {Object} Schema validation for field.
```

Comment:
(cc @gregrickaby ) here and elsewhere - are we using `@return` or `@returns`?

Reply:
This should probably be `@returns` and I'll change this since consistently, it's `@return`.